src/app/docs/toolbar/toolbar-docs.component.html

```diff
@@ -0,0 +1,147 @@
+      >
+      </sky-docs-design-guideline-thumbnail>
+
+      <sky-docs-design-guideline
+        heading="Standard toolbar actions"
+      >
+        <p>
+          Toolbars won't always need every possible button, but when present, buttons should be in the expected locations. Standard actions should appear from left to right in the following order when present in a toolbar:
+        </p>
+
+        <sky-toolbar-guidelines></sky-toolbar-guidelines>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Responsiveness"
+    >
+      <sky-docs-design-guideline
+        heading="Toolbar action wrapping"
+      >
+        <p>
+          The actions in a toolbar (including the more actions button) shouldn't extend beyond 50% of the screen. If the actions would extend beyond 50% of the screen, the rightmost action should collapse into the more actions menu.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't use toolbars with too many visible actions"
+          captionType="danger"
+          imageSource="~/assets/img/guidelines/toolbar/wrapping-dont-without-context-menu.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Do move the rightmost action into the more actions menu."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/toolbar/wrapping-do-with-context-menu.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline>
+        <p>
+          If a toolbar doesn't include a more actions menu and the actions extend beyond 50% of the screen, create a more actions menu for the two rightmost actions.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't add a more actions menu to toolbars with too many visible actions."
+          captionType="danger"
+          imageSource="~/assets/img/guidelines/toolbar/wrapping-dont-with-context-menu.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Do move the two rightmost actions into a more actions menu."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/toolbar/wrapping-do-without-context-menu.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Small form factors"
+      >
+        <p>
+          On small screens, action buttons should switch to icon-only buttons, the search field should collapse into a search icon, and the view switcher should collapse into a single icon. Buttons without icons should go in the more actions menu, and the more actions button, view switcher, and any other buttons with dropdowns should display their options when pressed.
```

Comment:
I'd change `should switch` to `switch`, both instances of `should collapse` to `collapses`, `should go` to `go`, and `should display` to `display`. And I think `pressed` should be `selected`.